Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,11 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function bmi(weight, height){
```

Comment:
If weight should always be in kg and height in m, how could you rename these two parameters to make that a bit clearer to someone else who might use your function in future?

Reply:
I changed the parameters name